JointMeasurements/Tests.qs

```diff
@@ -50,29 +50,26 @@ namespace Quantum.Kata.JointMeasurements {
             }
         }
         
-        AssertIntEqual(nOk, nTotal, $"{nTotal - nOk} test runs out of {nTotal} returned incorrect state.");
+        EqualityFactI(nOk, nTotal, $"{nTotal - nOk} test runs out of {nTotal} returned incorrect state.");
     }
     
     
     // ------------------------------------------------------
-    operation StatePrep_ParityMeasurement (qs : Qubit[], state : Int, alpha : Double) : Unit {
+    operation StatePrep_ParityMeasurement (qs : Qubit[], state : Int, alpha : Double) : Unit
+    is Adj {
         
-        body (...) {
-            // prep cos(alpha) * |0..0⟩ + sin(alpha) * |1..1⟩
-            Ry(2.0 * alpha, qs[0]);
-            for (i in 1 .. Length(qs) - 1) {
-                CNOT(qs[0], qs[i]);
-            }
+        // prep cos(alpha) * |0..0⟩ + sin(alpha) * |1..1⟩
+        Ry(2.0 * alpha, qs[0]);
+        for (i in 1 .. Length(qs) - 1) {
+            CNOT(qs[0], qs[i]);
```

Comment:
Minor note, but this can be written as `ApplyToEachA(CNOT(Head(qs), _), Rest(qs));`.

Reply:
There are plenty of things like that that could make use of the standard libraries that I did not address with this PR. This one was meant to do the bare minimum of updates related to breaking changes since we needed it for the release. I ended up also doing a little bit of refactoring to take advantage of new language features (couldn't help it), but I didn't do any refactoring related to leveraging the standard libraries. Let's make that a separate PR.